src/vs/workbench/contrib/extensions/browser/extensionsWorkbenchService.ts

```diff
@@ -792,6 +802,32 @@ export class ExtensionsWorkbenchService extends Disposable implements IExtension
 		this.telemetryService.publicLog2<InstalledExtensionsEvent, ExtensionsLoadClassification>('installedExtensions', { extensionIds: extensionIds.join(';'), count: extensionIds.length });
 	}
 
+	private updateRunningExtensions(): void {
+		this.extensionService.getExtensions().then(async runningExtensions => {
+			const deltaExtensionsDescription: IExtensionDescription[] = [];
+			deltaExtensionsDescription.push(...runningExtensions.filter(e => !this.runningExtensions?.includes(e)));
+			if (this.runningExtensions) {
+				deltaExtensionsDescription.push(...this.runningExtensions.filter(e => !runningExtensions.includes(e)));
+			}
+			this.runningExtensions = runningExtensions;
+
+			const installed = this.installed;
+			const deltaExtensions: IExtension[] = [];
+			const extsNotInstalled: IExtensionInfo[] = [];
+			for (const desc of deltaExtensionsDescription) {
+				const extension = installed.find(e => e.identifier.id === desc.id);
```

Comment:
Use `areSameExtensions()` util API to check if two extensions are same or not

Reply:
areSameExtensions() doesn't work here as `desc` is type IExtensionDescription